src/histolab/filters/image_filters.py

```diff
@@ -169,7 +169,8 @@ class Invert(ImageFilter):
     Example:
         >>> from PIL import Image
         >>> from histolab.filters.image_filters import Invert, RgbToGrayscale
-        >>> image_rgb = Image.open("path/to/image.png") # Read an RGB image
+        >>> from urllib.request import urlopen
+        >>> image_rgb = Image.open(urlopen("https://user-images.githubusercontent.com/4196091/92748324-5033e680-f385-11ea-812b-6a9a225ceca4.png")) # Read an RGB image
```

Comment:
can't we read from the tests/ folder

Reply:
yes i tried, but apparently we can't cause of the pythonpath
let's see if i can find a better way 🤷🏽‍♂️